bundles/cif-connector-graphql/src/main/java/com/adobe/cq/commerce/graphql/resource/CategoryResource.java

```diff
@@ -61,6 +61,7 @@
             } else {
                 map.put(LEAF_CATEGORY, false);
             }
+            map.put("hasChildren", category.getChildren().size() + category.getProductCount() > 0);
```

Comment:
You query for `children_count`, so that can be used here instead of `category.getChildren().size()`.

Reply:
That's what I thought too initially, but when I saw that getChildrenCount() (?) returns a String, I went for List.size()  which cannot be wrong.

Reply:
`hasChildren ` is used in several places, also in `CIFProductFieldHelper` class, should we introduce a constant of it?